skiko/src/awtMain/kotlin/org/jetbrains/skiko/swing/SwingOffscreenDrawer.kt

```diff
@@ -56,7 +58,13 @@ internal class SwingOffscreenDrawer(
         dirtyRectangles: List<Rectangle>
     ): BufferedImage {
         val src = ByteBuffer.wrap(bytes)
-        val image = BufferedImage(width, height, BufferedImage.TYPE_INT_ARGB_PRE)
+        if (bufferedImage == null || bufferedImage?.width != width || bufferedImage?.height != height) {
+            bufferedImage = BufferedImage(width, height, BufferedImage.TYPE_INT_ARGB_PRE)
+            bufferedImageGraphics = bufferedImage?.createGraphics()
+        }
+        val image = bufferedImage!!
+        bufferedImageGraphics?.clearRect(0,0, width, height)
```

Comment:
```suggestion
        } else {
            bufferedImageGraphics?.clearRect(0, 0, width, height)
        }
        val image = bufferedImage!!
```

It seems clearing is only required in case of re-use